portfolio/src/com.google.sps.FindMeetingQueryTest.txt

```diff
@@ -0,0 +1,4 @@
+-------------------------------------------------------------------------------
+Test set: com.google.sps.FindMeetingQueryTest
+-------------------------------------------------------------------------------
+Tests run: 11, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.13 sec
```

Comment:
I meant like adding the whole FindMeetingQueryTest.java file into the pull request.  Either move it into the same directory as your FindMeetingQuery.java, or move it somewhere closer.

Usually you'll be working on business logic and test code side by side.  So being able to easily find which test class addresses which business logic (eg. if they're in the same directory) would make it easier for maintenance.